backend/app.py

```
@@ -45,7 +45,7 @@ def price():
 
 @app.route("/calc")
 def calc():
-    expr = request.args.get("expr", "0")
+    expr = request.args.get("expr", "10")
     return jsonify(result=eval(expr))
```

Ruff (0.14.0)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)